README.md

```diff
@@ -937,12 +937,15 @@ constexpr void swap(static_vector<T, N>& x,
   noexcept(noexcept(x.swap(y)));
 ```
 
-> - _Constraints_: This function shall not participate in overload resolution
->   unless `is_swappable_v<T>` is `true`.
+> - _Mandates_: `is_swappable_v<T>` is `true` and `is_move_constructible_v<T>` is `true`.
 >
 > - _Effects_: As if by `x.swap(y)`.
```

Comment:
We don't use "as if by" in new wording since we now have "Equivalent to" [which has better magic](http://eel.is/c++draft/structure.specifications#4):

```suggestion
 > - _Effects_: Equivalent to `x.swap(y)`.
```